cypress/e2e/Transcript/Access/BoardAndDoBIsRequiredToSomeBoards.feature

```diff
@@ -0,0 +1,16 @@
+Feature: The user is required to complete their profile to see the transcripts
+  Background:
+    Given I am a registered user
+    And I have selected that I have a medical board
+    And I did not provide the required <MedicalBoardInformation>
```

Comment:
Please, don't use 1st person, to be consistent with the scenarios and other tests where we talk in the 3rd.